### ISSUE TITLE
delete twistd's "quiet" option

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3003]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3003|
|Type|task|
|Created|2008-01-20 15:51:13Z|

It doesn't do anything.

Attachments:

* [twistd-remove-quiet-flag](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/62a/62a227110055c937d62b8012bdd0769564870fe2/f7de3caab7ed698f3caefd80cf7df3f9074a5cc7) (590 bytes) - added by danderson on 2009-09-13 03:57:22Z - Patch for this issue.

<details><summary>Searchable metadata</summary>

```
trac-id__3003 3003
type__task task
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1200844273000000 1200844273000000
changetime__1252870292000000 1252870292000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="danderson's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|danderson set owner to danderson|
|-|-|

Here is a patch which fixes this issue. It's been 2 years since the desire to delete this has been present, I think that's long enough :).

$ trial twisted
...
Ran 5694 tests in 78.621s

PASSED (skips=628, expectedFailures=18, successes=5048)

**Comment by twisted-trac** [comment]
|<img alt="danderson's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|danderson removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Great, thanks. :)  Applying now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [27324]) Apply twistd-remove-quiet-flag

Author: danderson
Reviewer: exarkun
Fixes: [#3003](https://github.com/twisted/twisted/issues/3003)

Remove the `--quiet` option from `twistd` (a no-op for six years).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
